tee-worker/litentry/core/stf-task/receiver/src/handler/assertion.rs

```diff
@@ -82,7 +83,8 @@ where
 			Assertion::A7(min_balance) =>
 				build_holding_time(&self.req, AmountHoldingTimeType::DOT, min_balance),
 
-			// no need to pass `networks` again because it's the same as the `get_supported_web3networks`
+			// no need to pass `networks` again because it's the same as the
+			// `get_supported_web3networks`
```

Comment:
do you have special config with your 'cargo fmt'? Looks like line length changed.